DMTN-049.tex

```diff
@@ -656,14 +668,127 @@ \subsection{Considerations for Maximizing Early Science}\label{ssec:use_LOY1}
 Given the uncertainties on the actual construction schedule and commissioning period, several different plans for early science are being considered. 
```

Comment:
Add in a referecen here to the new Early Science document. rtn-011.lsst.io

Reply:
RTN-011 was being referenced in the previous sentence, so I moved it to this sentence, after "considered".